internal/deviations/deviations.go

```diff
@@ -65,8 +65,15 @@ package deviations
 
 import (
 	"flag"
+
+	"github.com/openconfig/ondatra"
 )
 
+// P4RTMissingDelete adds deviation for device that does not support delete mode in P4RT write requests.
+func P4RTMissingDelete(dut *ondatra.DUTDevice) bool {
+	return *p4rtMissingDelete
+}
+
 // Vendor deviation flags.
```

Comment:
Let's also expand this comment with a new paragraph that says all new flags should not be exported (be lowercase) and accessed from tests through a public accessors like those above.